composer/trainer/__init__.py

```diff
@@ -2,72 +2,66 @@
 
 """Train models!
 
-The trainer supports models with :class:`ComposerModel` instances.
-The :class:`Trainer` is highly customizable and can support a wide variety of workloads.
+The trainer supports models with :class:`~composer.models.base.ComposerModel` instances.
```

Comment:
can use :class:`.Trainer` throughout to make them more readable.

Reply:
Oh do you need the "." in front of the `Trainer`? It was originally written as :class:`Trainer` without the dot and the linking didn't work but I can fix